packages/svelteui-core/src/components/Notification/Notification.svelte

```diff
@@ -46,7 +47,7 @@
 >
 	{#if icon && !loading}
 		<slot class={classes.icon} name="icon">
```

Comment:
Could you remove this class from here and only apply below, or else it gives out weird behaviour. This was a nice catch on an unrelated bug 😅 
<img width="1463" alt="image" src="https://user-images.githubusercontent.com/25725586/203627505-308b3a58-4458-484d-999f-92a6d4c16717.png">

Reply:
done 👍🏻